apps/guardian-ui/src/components/SetConfiguration.tsx

```diff
@@ -122,6 +124,10 @@ export const SetConfiguration: React.FC<Props> = ({ next }: Props) => {
       )
     : Boolean(myName && password && hostServerUrl);
 
+  useEffect(() => {
+    setPasswordsMatch(password === confirmPassword);
+  }, [password, confirmPassword]);
+
```

Comment:
This doesn't need to be state / useEffect, `passwordsMatch` can be derived from the existing state via `password === confirmPassword` in the render method.

Reply:
fixed